ReleaseNotes-2.11

```diff
@@ -29,6 +29,8 @@ Features
 Generic:
  * You can track javascript errors that occur in the users browser with errbit, enable
    it in the API configuration file options.yml.
+ * Search was improved: searching for a just created or updated project or package will
```

Comment:
I suggest this:

`Search was improved: New and updated projects/packages are indexed directly instead of once every hour.`

Reply:
Thanks. Done.